bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java

```diff
@@ -511,8 +511,10 @@ public Void call() throws Exception {
                             ex.getMessage(), ex);
                 }
             } else {
-                logger.debug("Cannot notify handler about updated thing {}, because thing is not initialized "
-                        + "(must be in status ONLINE or OFFLINE).", thing.getThingTypeUID());
+                logger.debug(
+                        "Cannot notify handler about updated thing '{}', because thing is not initialized (must be in status ONLINE or OFFLINE). Start handler initialization instead.",
```

Comment:
Yes, you are right. I will change the message "... because the handler is not initialized .."